### DISCUSSION TITLE
Character-specific winner quotes

### DISCUSSION THREAD

**Discussion by LeoKids** [ikemen-engine/Ikemen-GO#1452] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

The problem is that the winning quotes although useful,doesn't have many options like if the character is fighting someone who is supposed to have a quote against it. A characters will retain its usual quotes regardless of who it is facing.

We do have alternate quotes for languages,but not the quotes we would want.

### Describe the solution you'd like

A solution if the character specific quotes (will probably be known as special quotes in code) will appear instead of the usual quotes whenever you face the opponent in a fight that has the specific quote with it.

For example with Kung Fu Man,his arch-nemesis is Suave Dude. His quote could be like this.

"You're gonna pay for messing with me and kidnapping my girlfriend! So i finally stopped you!"

In Ikemen,this could be coded like this (in CNS,of course,ZSS may differ)

specialvictory1 = "You're gonna pay for messing with me and kidnapping my girlfriend! So i finally stopped you!" (vs Suave Dude)

Keep in mind that this is a example. You can do whatever you want with it.

### Additional context

Play games like KOF 2002 and Street Fighter Alpha 3,You'll understand this.

### Search terms

quotes,victory

**Comment by Kokorz** [comment]
https://mugenguild.com/forum/topics/victoryquote-sctrl-170211.0.html

**Reply by Sejha** [reply]
> https://mugenguild.com/forum/topics/victoryquote-sctrl-170211.0.html

This is a nonmeaningful answer and ignores the actual request.

Id also appreciate this as a feature, especially considering how common it is these days.

**Reply by Kokorz** [reply, marked as answer]
This was so long ago admittedly and i shouldn't have linked to the sctrl with no explanation, but I can elaborate on why I linked to it.

If you use the victoryQuote state controller with the condition of checking for the opponent's name, you will get the exact same result.

For example, if Kung Fu Man's special quote to Suave Dude was assigned to victory30, you could then call victoryQuote anywhere in your code like so:

```
if p2, name = "Suave Dude" {
    VictoryQuote{value: 30}
}
```

OR (if you prefer CNS)

```
[State , Test]
type = VictoryQuote
trigger1 = p2, name = "Suave Dude"
value = 30
```

And the special victory quote would show if kfm wins against Suave Dude, making this proposal redundant. If you want to randomize from a selection of specific quotes for a character, you could use the randomRange trigger in the value. And if you even want it to be for a specific move, simply call the state controller in that move and change the value parameter.

I, however, think what would be nicer is room for more than 99 victory quotes.
